SafeguardCommon/Src/Modules/Error/CommonErrorCodes.pqm

```diff
@@ -4,6 +4,7 @@ let
         AuthenticationError = 4010,
         AuthorizationError = 4030,
         NotFound = 4040,
+        NotAcceptable = 4060,

```

Comment:
TestCommonRequestErrors.query.pq needs to be updated.

Reply:
Resolved by 

https://github.com/OneIdentity/SafeguardPowerBI/pull/68/commits/3eaa8e5f43b3de962ddfe30a6606694e7d6dfbe8